### PR TITLE
Adding inverse transform and adding dev transform

### PR DESCRIPTION
**Pull request by amir-zeraati**
Adding inverse transform function for transforming the output of trained neural network to the initial scale to be able to calculate errors like (MAE, MSE, RMSE).
Adding dev_transform function to transform the test set in the scale of training set to avoid data leakage.
these two functions use the .csv file that is returned by normal_seq function.

### CODE REVIEW

**Comment by organic-chemistry** [comment]
Maybe it could be good in the csvs file to put a column with the name of the signals just to check it afterword.
